app/screens/AddContact.js

```diff
@@ -32,15 +41,7 @@ export default class AddContact extends Component {
 
   resetContactState() {
     Actions.contactList();
-    this.setState({
-      firstName: '',
-      lastName: '',
-      birthdayDate: new Date(),
-      avatar: null
-    });
-  }
-
-  componentWillUnmount() {
+    Actions.pop();
```

Comment:
I'm glad you found that useful! `type: reset` wasn't working for me either, but the pop() and push() methods apparently work with the react-native-router-flux too!